doc/en/example/pythoncollection.rst

```diff
@@ -206,7 +206,7 @@ You can always peek at the collection tree without running tests like this:
 
 .. code-block:: pytest
 
-    . $ pytest --collect-only pythoncollection.py
```

Comment:
They are there to *not* run regendoc, because regendoc uses the line starting as `$` to mean to execute the shell command:

https://github.com/pytest-dev/regendoc/blob/27e3652723659d8b2d7a5311567480fccc49cbe2/regendoc/parse.py#L82-L84

Reply:
So this PR is probably bogus. Should I open an issue to have a mechanism for this that doesn't uglify the docs?

Reply:
Good idea, I would kindly ask to create it in https://github.com/pytest-dev/regendoc though. 👍

Reply:
https://github.com/pytest-dev/regendoc/issues/9

Reply:
Note that studying the code, a dot with a space should not stop regendoc:

https://github.com/pytest-dev/regendoc/blob/27e3652723659d8b2d7a5311567480fccc49cbe2/regendoc/parse.py#L85-L88

Reply:
> They are there to _not_ run regendoc

Why though?  Shouldn't we have working examples for those also?

Reply:
You are right, probably they should